data/strings/strings.txt

```diff
@@ -22527,7 +22527,7 @@
     zh-Hant = 使用條款
 
   [button_layer_traffic]
-    tags = android,ios
+    tags = android
```

Comment:
Does it mean that Android still has a traffic button somewhere? Can/should it also be removed?
CC @Jean-BaptisteC

Reply:
Button is not present on Android but code base was keep

Reply:
Ok I thought I remove the code. It can always be recovered from git. But you guys decide how you want it.

Reply:
@biodranik I think this is ready to merge